lib/parse.js

```diff
@@ -8,6 +8,9 @@ function number(string) {
   if (string === '') {
     return 0;
   }
+  if (string === undefined) {
+    return 0;
+  }
```

Comment:
So I'm not sure about this. I feel undefined numbers are an indication of failure -- the scraper found nothing, so it should stop and we should report an error. @qgolsteyn thoughts?

Reply:
actually that's exactly what it was. I added it when debugging OH. We can remove it, you're not wrong.